dymos/phase/phase.py

```diff
@@ -1083,9 +1083,9 @@ def add_boundary_constraint(self, name, loc, constraint_name=None, units=None,
         loc : str
             The location of the boundary constraint ('initial' or 'final').
         constraint_name : str or None
-            The name of the variable as provided to the boundary constraint comp.  By
-            default this is the last element in `name` when split by dots.  The user may
-            override the constraint name if splitting the path causes name collisions.
+            The name of the boundary constraint. By default, this is the left-hand side of
```

Comment:
Reword: "By default, this is 'var_constraint' if name is a single variable, or the left-hand side of the equation if name is an expression."